src/waypoints/route.py

```diff
@@ -163,140 +153,134 @@ def route(start_pos, end_pos, filepath):
     status, route = router.doRoute(start, end)  # Find the route - a list of OSM nodes
 
     if status == 'success':
-        routeLatLons = list(map(router.nodeLatLon, route))  # Get actual`` route coordinates
+        route_lat_lons = list(
+            map(router.nodeLatLon, route)
+        )  # Get actual route coordinates
     else:
-        routeLatLons = []
+        route_lat_lons = []
 
-    return routeLatLons, (status == 'success')
+    return route_lat_lons, (status == 'success')
 
 
-# FIXME: this is a really shitty algorithm
 def add_nodes_to_route(route, max_dist):
-
     """
-    Added additional nodes to `route` (a of list of tuples (lat, lon)) such that
-    the distance between any two nodes in the route is less than or equivalent
-    to `max_dist`.
-
-    The current implementation simply loops through each pair of points in the
-    original route and adds point a distance `max_dist` away from the first
-    point in the pair until the added point is within `max_dist` of the second
-    point in the pair.
+    Add additional nodes to `route` so that the distance between
+    any two nodes is less than or equal to `max_dist`.
     """
 
     def node_distance(node_a, node_b):
         return math.dist(node_a, node_b)
 
     i = 0
     while i < len(route) - 1:
-
         node_a = route[i]
         node_b = route[i + 1]
-
         dist = node_distance(node_a, node_b)
-
-        # start at node_a and iterate until we are within max_dist of node_b
```

Comment:
I'm assuming that you're deleting all the comments to make it more in line with the rest of the code being uncommented, but I think there's no harm in explaining what you're doing

Reply:
did not mean to delete these, will fix